bell.js

```diff
@@ -17,7 +17,7 @@ const URL = require('url');
 process.on('SIGINT', function () {
     console.log("Caught interrupt signal");
     console.log("[INFO] Going offline");
-    sensorService.kill();
+//    sensorService.kill();
```

Comment:
Die moeten we even configurable maken.

Reply:
```suggestion
//    if (sensorservice !== 'undefined'){sensorService.kill();}
```
Alleen als hij überhaupt draait killen. Optie?